File: src/libs/ThirdParty.VBSuppliers/Properties/ExpediaRapid/SerializableClasses/Book/Address.vb
Imports Newtonsoft.Json

Namespace ExpediaRapid.SerializableClasses.Book
    Public Class Address

        <JsonProperty("line_1")>
        Public Property Line1 As String

        <JsonProperty("line_2")>
        Public Property Line2 As String

        <JsonProperty("line_3")>
        Public Property Line3 As String

        <JsonProperty("city")>
        Public Property City As String

        <JsonProperty("state_province_code")>
        Public Property StateProvinceCode As String

        <JsonProperty("postal_code")>
        Public Property PostalCode As String

        <JsonProperty("country_code")>
        Public Property CountryCode As String

    End Class

End Namespace

File: src/libs/iVectorOne.Suppliers/ExpediaRapid/SerializableClasses/Book/Address.cs
namespace ThirdParty.CSSuppliers.ExpediaRapid.SerializableClasses.Book
{
    using Newtonsoft.Json;

    public class Address
    {

        [JsonProperty("line_1")]
        public string Line1 { get; set; }

        [JsonProperty("line_2")]
        public string Line2 { get; set; }

        [JsonProperty("line_3")]
        public string Line3 { get; set; }

        [JsonProperty("city")]
        public string City { get; set; }

        [JsonProperty("state_province_code")]
        public string StateProvinceCode { get; set; }

        [JsonProperty("postal_code")]
        public string PostalCode { get; set; }

        [JsonProperty("country_code")]
        public string CountryCode { get; set; }

    }

}